src/common/image.cpp

```diff
@@ -2185,6 +2185,9 @@ unsigned char *wxImage::GetAlpha() const
 
 void wxImage::InitAlpha()
 {
+    if (!IsOk())
```

Comment:
As calling basically any method on an invalid `wxImage` is a programmer mistake, I wonder if `wxCHECK()` would not be better here.

Reply:
That makes sense -- updated accordingly, copying the same check from the SetAlpha() function above this one.